### PR TITLE
Step 3 of error flow

### PR DESCRIPTION
**Pull request by easternbloc**
* New personal-details-brp page
* If we have two types of pages that are similar we should keep the naming consistent and just use the template name in the steps

### CODE REVIEW

File: routes/steps/error.js [hunk 1]
```diff
@@ -33,10 +33,23 @@ module.exports = {
...
+    template: 'personal-details-brp.html',
+    fields: [
+      'fullname',
+      'date-of-birth-day',
+      'date-of-birth-month',
+      'date-of-birth-year',
+      'nationality',
+      'passport'
+    ],
     next: '/confirmation'
```
Comment (JoeChapman) [inline routes/steps/error.js]: '/contact-details'

File: views/pages/personal-details-passport.html [hunk 2]
```diff
@@ -4,10 +4,10 @@
   {{> partials-navigation}}
 {{/propositionHeader}}
 
-{{$header}}{{#t}}steps.three.header{{/t}}{{/header}}
+{{$header}}{{#t}}steps.personal-details.header{{/t}}{{/header}}
```
Comment (JoeChapman) [inline views/pages/personal-details-passport.html]: replace `steps` with `pages`

File: views/pages/personal-details-passport.html [hunk 3]
```diff
@@ -4,10 +4,10 @@
...
 
 {{$content}}
-  <p>{{#t}}steps.three.subheader{{/t}}</p>
+  <p>{{#t}}steps.personal-details.subheader{{/t}}</p>
```
Comment (JoeChapman) [inline views/pages/personal-details-passport.html]: replace `steps` with `pages`

File: views/pages/personal-details-passport.html (hunk 2 above)
Comment (easternbloc) [inline views/pages/personal-details-passport.html]: (ノಠ益ಠ)ノ彡

**Comment by JoeChapman** [comment]
reeeeeeeeeeeeeeeeebase

**Pull request merged**